roles/container-engine/docker/defaults/main.yml

```diff
@@ -1,7 +1,6 @@
 ---
 docker_version: '19.03'
-docker_cli_version: "{{ 'latest' if docker_version != 'latest' and docker_version is version('18.09', '<') else docker_version }}"
-docker_selinux_version: '17.03'
```

Comment:
just NOTE: This PR removes support of older versions of docker. Then we can make here simple like this.